library/pages/components/ExperienceExamplesPage/index.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { Prismic, QueryAt, QueryMulti } from 'react-prismic-cms';
+import ExperienceExample from './experienceExample';
+import './styles.css';
+
+export default function ExperienceExamplesPage() {
+    return (
+        <Prismic repo="rainbow-doc">
+            <div className="react-rainbow-experience-examples-view-port">
+                <h1 className="react-rainbow-experience-examples-header-title">
+                    Experience examples
+                </h1>
+                <div className="react-rainbow-experience-examples-container">
+                    <QueryMulti component={ExperienceExample} type="experience-examples">
```

Comment:
here since only one query is used (QueryAt), the QueryMulti is not needed. QueryMulti is only used when multiples queries are passed:
```
<QueryMulti component={Util} type={value} componentName={name}>
    <QueryAt path="document.type" value={value} />
    <QueryAt path="document.tags" value={[name]} />
</QueryMulti>
```